.github/workflows/prune_stale_issues.yml

```diff
@@ -0,0 +1,15 @@
+name: "Close stale issues"
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v1.1.0
+      with:
+        repo-token: ${{ secrets.vsphere_github_actions }}
```

Comment:
Is this the right repo-token for the kubernetes repo?

Reply:
Great catch, I'll fix that